site/gatsby-site/src/utils/AlgoliaUpdater.js

```diff
@@ -15,11 +17,36 @@ const truncate = (doc) => {
   return doc;
 };
 
+const classificationsWhitelist = [
+  'Harm Distribution Basis',
+  'Intent',
+  'Lives Lost',
+  'Location',
+  'Named Entities',
+  'Near Miss',
+  'Severity',
+  'AI Applications',
+  'AI Techniques',
+  'Financial Cost',
+  'Harm Type',
+  'Infrastructure Sectors',
+  'Level of Autonomy',
+  'Lives Lost',
+  'Nature of End User',
+  'Physical System',
+  'Problem Nature',
+  'Public Sector Deployment',
+  'Relevant AI functions',
+  'Sector of Deployment',
+  'System Developer',
+  'Technology Purveyor',
+];
+
 const getClassificationArray = (cObj, namespace) => {
   const cArray = [];
 
   for (const c in cObj) {
-    if (cObj[c] !== null) {
+    if (cObj[c] !== null && classificationsWhitelist.includes(c)) {
```

Comment:
Under what condition do we get classifications not in the whitelist?

Reply:
There are a few like `Annotator` that shouldn't be sent to Algolia.